injector/LogInjector.py

```
@@ -1,11 +1,12 @@
 import ast
 import json
 from injector.helper import getVarLogStmt, getLtLogStmt, getAssignStmt, getAdliConfiguration, getEncodedOutputStmt, getEmptyRootNode, getUniqueIdAssignStmt, getRootUidAssign
-from injector.helper import injectRootLoggingSetup, injectLoggingSetup, getTag
+from injector.helper import injectRootLoggingSetup, injectLoggingSetup, getTag, getGlobalVarsLogStmt, getLocalVarsLogStmt
```

Ruff (0.11.9)

`injector.helper.getGlobalVarsLogStmt` imported but unused

Remove unused import: `injector.helper.getGlobalVarsLogStmt`

(F401)